ui/test/nuxeo-video-info.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, flush, html } from '@nuxeo/nuxeo-elements/test/test-helpers';
+import '../nuxeo-video/nuxeo-video-info.js';
+import videoProperties from './resources/videoProperties';
+
+suite('nuxeo-video-info', () => {
+  let element;
+
+  const document = {
+    properties: {
+      'vid:info': videoProperties['vid:info'],
+    },
+  };
+
+  setup(async () => {
+    element = await fixture(html`
+      <nuxeo-video-info .document="${document}"></nuxeo-video-info>
+    `);
+    await flush();
+  });
+
+  test('Should display 5 slots for video info', () => {
```

Comment:
Looking the other test class, what about naming this test `Should display 5 properties when video contains info`?